core/interfaces/i_ast_node_location_svg.ts

```diff
@@ -11,6 +11,4 @@ import type {IASTNodeLocation} from './i_ast_node_location.js';
 /**
  * An AST node location SVG interface.
  */
-export interface IASTNodeLocationSvg extends IASTNodeLocation {
-
-}
+export interface IASTNodeLocationSvg extends IASTNodeLocation {}
```

Comment:
Could this interface not be deleted entirely, now that it is empty?

I suppose we could formally deprecate it first, but since anyone who was calling `setFooSvg` on an `IASTNodeLocationSvg` instance already has broken code I'm not sure it's worth it.

Reply:
Filed as https://github.com/google/blockly/issues/9006